### PR TITLE
add pt-br

### PR DESCRIPTION
**Pull request by jmmander**
Screenshot 2023-09-27 at 12 40 58 PM

### CODE REVIEW

File: guides/localization.mdx [hunk 1]
```diff
@@ -90,6 +90,7 @@ We support translations into these languages for now:
 | French   | fr     |
 | German   | de     |
 | Spanish  | es     |
+| Brazillian Portugese  | pt-BR     |
```
Comment (brentkulwicki) [inline guides/localization.mdx:93]: Check me but I think this is just one L in Brazilian right?

Comment (brentkulwicki) [reply guides/localization.mdx:93]: Also I believe Portuguese is spelled wrong

Comment (jmmander) [reply guides/localization.mdx:93]: oops

Comment (jmmander) [reply guides/localization.mdx]: fixed and thanks!

**Review by brentkulwicki — APPROVED** [review]

**Pull request merged**